doc/method/method.md

```diff
@@ -1,185 +1,456 @@
+showed up as direct children of Mammalia.  While the NCBI Taxonomy
+usually gives a modern classification for the species it covers, it
+makes no attempt to fit in extinct taxa, so when such taxa are found
+in other taxonomies, there is no good place to put them in the Open
+Tree taxonomy.  Filtering out extinct taxa leads to a cleaner result.
+
+In two cases, an imported source taxonomy is split into parts before
+processing continues, so that the parts can be positioned in different
+places in the align/merge order.  The decision to divide a source
+taxonomy is made by a curator.  In the case of Index Fungorum, a
+curator noted that the coverage and curation quality outside of Fungi
+did not seem to improve on NCBI Taxonomy, while inside Fungi it did.
+Index Fungorum is therefore split into high priority Fungi and lower
+priority everything-except-Fungi.  Similarly, in the case of WoRMS,
+Malacostraca was a project priority at that point, and the quality of
+the rest of WoRMS was unreviewed, so rather than risk possible
+problems by overriding NCBI, we simply made non-Malaconstraca be low
+priority.
+
+After each source taxonomy is loaded, the following two normalizations
+are performed:
+
+ 1. Child taxa of "containers" in the source taxonomy are made to be
+    children of the parent container's parent.  "Containers" are
+    groupings in the tree that don't represent taxa, for example
+    "incertae sedis" or "environmental samples" nodes.  The members of
+    a container aren't more closely related to one another than they
+    are to the container's siblings; the container is only present as
+    a way to say something about the members.  The fact that a taxon
+    had originally been in a container is recorded as a flag on the
+    child node.
  1. Monotypic homonym removal - when taxon with name N has as its
-    only child another taxon with name N, remove one of the two
-
-Before alignment and merge, each source taxonomy gets patched
-individually.  It's always best to treat a problem as early as
-possible, so that its ill effects don't interfere with alignment of
-other taxonomies and with proper synthesis.  This is separate from the
-general patch phase that takes place at the end of taxonomy
-construction (below). A frequent kind of patch is to add a synonym or
-change a name so that a source taxon aligns with a taxon from an
-earlier or later source.  Patches that bring a source taxonomy into
-agreement with the skeleton also happen here.
-
-## Separation
+    only child another taxon with name N, the parent is removed.
+    This is done to avoid an ambiguity when later on a node with name
+    N needs to be matched.
+
+## Source patching
+
+Most source taxonomies have scripted ad hoc patches that are applied
+at this point.  Some capitalizations and misspellings are fixed, and
+synonyms added (e.g. 'Stramenopiles' for 'Heterokonta'), to improve
+matches between taxonomies.  Other patches, such as extinct or extant
```

Comment:
some information (that ends up in OTT) comes from source taxonomies, other information comes from patches. so a taxon can be extinct either because the source taxonomy says so or because a patch says so.  does that need to be said?